File: web/src/api/admin/gost_node_logger.js
import request from "../../request/index.js";

const baseUrl = '/v1/admin/gost/node/logger'

export const apiAdminGostNodeLoggerPage = (data) => {
    return request.request({
        url: `${baseUrl}/page`,
        method: 'POST',
        data
    })
}

File: server/router/admin/gost_node_logger.go
package admin

import (
	"github.com/gin-gonic/gin"
	"server/controller/admin/gost_node_logger"
	"server/global"
	"server/router/middleware"
)

func InitGostNodeLogger(group *gin.RouterGroup) {
	g := group.Group("gost/node/logger", middleware.Auth(global.Jwt))
	g.POST("page", gost_node_logger.Page)
}
